### PR TITLE
Update file permissions/ownership/group bash template to better support "file_regex" parameter

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Update file permissions/ownership/group bash template to better support `file_regex` parameter.
- It also enables back again bash remediation for `file_permissions_master_openvswitch`.

#### Rationale:

- Now the `file_regex` applies to basename of the file instead of what's is outputed from `find`.

TODO:

- [x] New template documentation.

### CODE REVIEW

**Comment by ggbecker** [comment]
@rmetrich this might interest you.

File: shared/templates/template_BASH_file_groupowner [hunk 1]
```diff
@@ -5,7 +5,12 @@
 # disruption = low
 
 {{% if IS_DIRECTORY and FILE_REGEX %}}
-find {{{ FILEPATH }}} -regex '{{{ FILE_REGEX }}}' -exec chgrp {{{ FILEGID }}} {} \;
+readarray -t files < <(find {{{ FILEPATH }}})
+for file in ${files[@]}; do
```
Comment (jan-cerny) [inline shared/templates/template_BASH_file_groupowner:9]:
```suggestion
for file in "${files[@]}"; do
```

Comment (ggbecker) [reply shared/templates/template_BASH_file_groupowner]: Fixed

**Comment by ggbecker** [comment]
Note to documentation was added. Ready for review.

**Comment by jan-cerny** [comment]
LGTM

**Pull request merged**